build.gradle

```diff
@@ -41,6 +45,9 @@ dependencies {
     // Dependency injection
     compile 'com.google.inject:guice:4.0'
 
+    // Plugin meta
+    compile 'org.spongepowered:plugin-meta:0.1-SNAPSHOT'
```

Comment:
@liach 

Read the entire PR in the future before issuing replies that waste everyone's time.